reflex/app.py

```diff
@@ -586,6 +590,8 @@ async def upload_file(files: List[UploadFile]):
             assert file.filename is not None
             file.filename = file.filename.split(":")[-1]
 
+        # get the current session ID
+        sid = app.session_map[token]
         # Get the state for the session.
         state = app.state_manager.get_state(token)
```

Comment:
i thought the state knew the `sid` already...

```
sid = state.get_sid()
```

Reply:
ah you're absolutely right, that should make the session mapping redundant.